src/pymor/models/transforms.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from scipy.linalg import inv, null_space
+
+from pymor.models.interface import Model
+
+
+class MoebiusTransform(Model):
```

Comment:
```suggestion
from pymor.core.base import ImmutableObject


class MoebiusTransform(ImmutableObject):
```

It is not really a model, similar to [`TransferFunction`](https://github.com/pymor/pymor/blob/b5eb4e4b27f5a2028776e401e1813792e2044d89/src/pymor/models/transfer_function.py#L15).